api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt

```diff
@@ -0,0 +1,44 @@
+package com.openlattice.organizations
+
+import retrofit2.http.*
+import java.util.*
+
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+interface DatasourcesApi {
+    companion object {
+        // @formatter:off
+        const val SERVICE = "/datastore"
+        const val CONTROLLER = "/organization"
```

Comment:
We have `/datastore/organizations` and with this `/datastore/organization`. Any reason why these have to be named so similarly? Why isn't the controller `/datasources`?

Reply:
Ignore this.